app/views/potential_match_settings/_potential_match_settings_form.html.erb

```diff
@@ -36,15 +36,17 @@
         <dd>
           <fieldset>
             <ul>
-            <% TagSet::TAG_TYPES.map {|t| t.pluralize }.each do |type| %>
-            <li class="<%= type.downcase %>"><%= match_settings_form.check_box("include_optional_#{type}", :checked => potential_match_settings.send("include_optional_#{type}")) %>
-          <%= match_settings_form.label("include_optional_#{type}", type.capitalize) %></li>
-      <% end %>
-      </ul>
+            <% TagSet::TAG_TYPES.map.each do |type| %>
+              <li class="<%= TagSet.find_type_label(type) %>">
+                <%= match_settings_form.check_box("include_optional_#{type.pluralize}", :checked => potential_match_settings.send("include_optional_#{type.pluralize}")) %>
+                <%= match_settings_form.label("include_optional_#{type.pluralize}", t("tag_types.#{type}").capitalize) %>
```

Comment:
Comparing this to production, I think we need `pluralize` on `t("tag_types.#{type}").capitalize` as well. (If you can also make this say "Archive Warnings" to match the rest of the form, that would be great, but it's not crucial.)